### ISSUE TITLE
Beast v84: Bad hash

### ISSUE THREAD

**Issue by jmin911** [Microsoft/vcpkg#1527]
Run ".\vcpkg install beast:x86-windows-static"

Error:
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:47 (message):

  File does not have expected hash:

          File path: [ C:/src/vcpkg/downloads/vinniefalco-Beast-32902fad8b7c57be27b96e00ed6ec82748732133.tar.gz ]
      Expected hash: [ 55d1ccff105f997ab7b607528980161baf22aff56109e368a6232f3cfe1fbbf3f060e2b88d68f00728e75b951b60291b8cd6d56a3e299208e6cd757cb53bd774 ]
        Actual hash: [ 409bd23e19c4bb157619405713f73b90c5f680bec487ad7b6fc8bcd30db54444a78950ce631c8e267e4397b9128679528fd237d3c53f7e1e55634473a169af5f ]

**Comment by jasjuang** [comment]
@vinniefalco should be able to help you

**Comment by vinniefalco** [comment]
Investigating...

**Comment by vinniefalco** [comment]
I cannot reproduce the problem. I am using the tip of the vcpkg **master** branch:
```
$ git log -1
commit 6167932bb60deb76f34669d57a136247014d622d
Merge: e2cdeddd 28ea7814
Author: Robert Schumacher <roschuma@microsoft.com>
Date:   Tue Jul 25 10:13:02 2017 -0700

    Merge pull request #1532 from Mixaill/freerdp-20rc1

    [freerdp] [SECURITY] update to 2.0-rc0
```

The output of `vcpkg install beast` is as follows:
```
The following packages will be built and installed:
    beast:x86-windows
Building package beast:x86-windows... 
-- CURRENT_INSTALLED_DIR=C:/Users/vinnie/src/vcpkg/installed/x86-windows
-- DOWNLOADS=C:/Users/vinnie/src/vcpkg/downloads
-- CURRENT_PACKAGES_DIR=C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows
-- CURRENT_BUILDTREES_DIR=C:/Users/vinnie/src/vcpkg/buildtrees/beast
-- CURRENT_PORT_DIR=C:/Users/vinnie/src/vcpkg/ports/beast/.
-- Using cached C:/Users/vinnie/src/vcpkg/downloads/vinniefalco-Beast-32902fad8b7c57be27b96e00ed6ec82748732133.tar.gz
-- Testing integrity of cached file...
-- Testing integrity of cached file... OK
-- Extracting done
-- Up-to-date: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/config.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/async_result.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/bind_handler.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/buffered_read_stream.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/buffers_adapter.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/buffer_cat.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/buffer_prefix.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/consuming_buffers.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/base64.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/bind_handler.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/buffers_ref.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/clamp.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/config.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/cpu_info.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/empty_base_optimization.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/integer_sequence.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/in_place_init.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/ostream.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/sha1.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/static_ostream.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/static_string.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/type_traits.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/detail/varint.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/drain_buffer.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/error.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/file.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/file_base.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/file_posix.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/file_stdio.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/file_win32.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/flat_buffer.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/flat_static_buffer.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/handler_alloc.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/handler_ptr.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/buffered_read_stream.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/buffers_adapter.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/buffer_cat.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/buffer_prefix.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/consuming_buffers.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/file_posix.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/file_stdio.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/file_win32.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/flat_buffer.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/flat_static_buffer.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/handler_ptr.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/multi_buffer.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/read_size.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/static_buffer.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/static_string.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/impl/string_param.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/multi_buffer.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/ostream.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/read_size.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/span.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/static_buffer.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/static_string.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/string.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/string_param.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core/type_traits.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/core.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/basic_dynamic_body.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/basic_file_body.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/basic_parser.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/buffer_body.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/chunk_encode.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/detail
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/detail/basic_parsed_list.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/detail/basic_parser.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/detail/chunk_encode.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/detail/rfc7230.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/detail/type_traits.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/dynamic_body.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/empty_body.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/error.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/field.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/fields.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/file_body.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/basic_parser.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/chunk_encode.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/error.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/field.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/fields.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/file_body_win32.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/message.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/parser.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/read.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/rfc7230.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/serializer.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/status.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/verb.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/impl/write.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/message.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/parser.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/read.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/rfc7230.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/serializer.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/span_body.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/status.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/string_body.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/type_traits.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/vector_body.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/verb.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http/write.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/http.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/version.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/detail
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/detail/frame.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/detail/hybi13.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/detail/mask.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/detail/pausation.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/detail/pmd_extension.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/detail/type_traits.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/detail/utf8_checker.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/error.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/impl
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/impl/accept.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/impl/close.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/impl/error.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/impl/fail.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/impl/handshake.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/impl/ping.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/impl/read.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/impl/rfc6455.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/impl/ssl.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/impl/stream.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/impl/teardown.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/impl/write.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/option.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/rfc6455.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/ssl.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/stream.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket/teardown.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/websocket.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib/deflate_stream.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib/detail
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib/detail/bitstream.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib/detail/deflate_stream.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib/detail/inflate_stream.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib/detail/ranges.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib/detail/window.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib/error.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib/impl
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib/impl/error.ipp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib/inflate_stream.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib/zlib.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast/zlib.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/include/beast.hpp
-- Installing: C:/Users/vinnie/src/vcpkg/packages/beast_x86-windows/share/beast/copyright
-- Performing post-build validation
-- Performing post-build validation done
Building package beast:x86-windows... done
Installing package beast:x86-windows... 
Installing package beast:x86-windows... done
```

Please try removing and reinstalling your package, after clearing any cached items.

**Comment by ras0219-msft** [comment]
@vinniefalco @jmin911 

Just to add a bit of clarity: we cache downloads (even incorrect ones) inside `downloads\`. If you've previously successfully or unsuccessfully downloaded the archive, it won't re-download it unless you explicitly delete it.

**Comment by jasjuang** [comment]
@vinniefalco I think he is having trouble with building the static version. `vcpkg install beast:x86-windows-static`

**Comment by ras0219-msft** [comment]
@vinniefalco I'm able to reproduce this issue. Make sure you delete the downloaded file and re-try!

  File does not have expected hash:

          File path: [ C:/src/vcpkg/downloads/vinniefalco-Beast-1bc30cb6e4ce0fd2e288b2cd74d18f857801cbc7.tar.gz ]
      Expected hash: [ c3d6b99273c2b2acdd366547119e78c25f788dabfae11f533aea8d31d03383b46516398370103b01e3b9bb0bb3921c981299cd3591107845630181b48e3b010e ]
        Actual hash: [ 2d2e2f8431969f734684cc92686fcc6577e4b70c6d27a0bf583fe2da17f2cf47f0b4136652cdfcfacba540c687abbb34febd0e1ff6c82a02486c36a8648d8d80 ]

@jasjuang Since this is a download failure, it's independent of the triplet.

I'm going to push an update to the hash.

**Comment by vinniefalco** [comment]
Beast is header-only so I'm not sure what the "static version" means

**Comment by ras0219-msft** [comment]
Since I'm getting the same hash as @jmin911, I'll push a portfile update to correct this.

**Comment by ras0219-msft** [comment]
@jmin911 Should be fixed by 4ae13d40

@vinniefalco You may need to run
```
rm C:/Users/vinnie/src/vcpkg/downloads/vinniefalco-Beast-32902fad8b7c57be27b96e00ed6ec82748732133.tar.gz
```
if it complains about mismatching hashes.
